Sources/CreateAPI/Generator/Generator+Schemas.swift

```diff
@@ -337,7 +337,16 @@ extension Generator {
     }
 
     private func makeInlineProperties(for type: TypeName, object: JSONSchema.ObjectContext, context: Context) throws -> [Property] {
-        var keys = object.properties.keys
+        let excludedProperties = options.entities.exclude
+            .filter { $0.name == type.rawValue }
+            .compactMap { $0.property }
+        let excludedDiff = Set(excludedProperties).subtracting(object.properties.keys)
+        
+        for diff in excludedDiff {
+            try handle(warning: "Excluded property \(type.rawValue).\(diff) does not exist on schema object \(type.rawValue)")
```

Comment:
I'm just wondering if this could be a bit more explanatory? 

```diff
- Excluded property User.id does not exist on schema object User
+ Invalid entity exclude 'User.id'. Property 'id' does not exist on schema 'User'
```

```suggestion
            try handle(warning: "Invalid entity exclude '\(type.rawValue).\(diff)'. Property '\(diff) does not exist on schema '\(type.rawValue)'")
```